core/domain/exp_services.py

```diff
@@ -3963,3 +3978,22 @@ def rollback_exploration_to_safe_state(exp_id: str) -> int:
         caching_services.delete_multi(
             caching_services.CACHE_NAMESPACE_EXPLORATION, None, [exp_id])
     return last_known_safe_version
+
+
+def is_voiceover_allowed_for_exploration(exploration_id: str) -> bool:
```

Comment:
Let's call this does_exploration_support_voiceovers(), it's a bit cleaner.

Reply:
Done